onnx/defs/math/defs.cc

```diff
@@ -54,7 +52,8 @@ case where axis=1, this means the input tensor will be coerced into a 2D tensor
 of dimensions [a_0, a_1 * ... * a_{n-1}], where a_0 is often the batch size.
 In this situation, we must have a_0 = N and a_1 * ... * a_{n-1} = D.
 Each of these dimensions must be matched correctly, or else the operator
-will throw errors.
+will throw errors. The output tensor has the same shape
+and contains the {name} values of the corresponding input.
```

Comment:
What does `{name}` mean?

Reply:
It is replaced by the value of the parameter name, as in two lines below.

Reply:
I moved this line from the top to the bottom where I thought it would be meaningful. Will fix this.

Reply:
This line is from above. Please check.

Reply:
It should be okay to leave as it is. This doc generator is shared between Softmax, Hardmax, LogSoftmax and the name is replaced in the generated doc.